hawkbit-core/src/main/java/org/eclipse/hawkbit/api/APIType.java

```diff
@@ -9,8 +9,18 @@
 package org.eclipse.hawkbit.api;
 
 /**
- * Represented the supported protocols for artifact url's.
+ * Type of hawkBit API that
```

Comment:
> that

Look's like either second part of the sentence  is missing or `that` is too much